mpi-proxy-split/lower-half/static_libs.txt

```diff
@@ -1 +1 @@
--lmpi_gnu_91 -lpmi -lpmi2 /opt/cray/libfabric/1.15.0.0/lib64/libfabric.a -lcxi -L/global/homes/z/zz217/local/lib -ljson-c  -lcurl  -lssl -lcrypto -lz -latomic -lpthread /opt/cray/pe/pals/1.1.8/lib/libpals.a -Wl,--end-group
+-lpmi -lpmi2 /opt/cray/libfabric/1.15.0.0/lib64/libfabric.a -lcxi -L/global/homes/z/zz217/local/lib -ljson-c  -lcurl  -lssl -lcrypto -lz -latomic -lpthread /opt/cray/pe/pals/1.1.8/lib/libpals.a -Wl,--end-group
```

Comment:
Here also, please remove the `-Wl,--end-group` at the end, and add it back into any Makefile that uese `static_libs.txt` from lower-half subdirectory.

Reply:
Done.